### DISCUSSION TITLE
Make deduplication a bit smarter

### DISCUSSION THREAD

**Discussion by Schmidsfeld** [stashapp/stash#3358] [Ideas]
In the Tools section there is the verry usefull tool of finding duplicated scenes and deleting some of them (to free up disk space and remove clutter). This method currently only uses phashes as a criteria.

I propose to improve this by adding two more strategies:

Multi file scenes:
Show also duplicated files - lately scenes are sometimes sorted into combined scenes where more than one scene is attached to it. I propose to simply add them as a criteria - similar to the filter tag (but with the more usefull interface of the duplicate scene detector)

Same StashID:
Sometimes the scene is to viible different from a clone. Most often if you have a significantly longer version fo the same scene. Luckyly quite often both phashes are in the stashdb. If you could filter for the same stashID these scenes would also be caught.

**Comment by Snooch2dabooch** [comment]
Sorry to bump an old thread, but I suppose It's better than creating a duplicate?
+1 to this.. also, In the dropdown box.. I would like "Select Every File In Each Duplicated Group, Except The Smallest File"  - for those of us that dont mind lower res to save disk space (or maybe we have converted to hevc/av1 to save disk.)

**Comment by swoop124** [comment]
Hello,

i think this is a good idea.
i habe multiple scenes with more than one File. 
So it would be great if i could delete thoes duplicated files with a klick of a button. Maybe with a filter to delete only older files or something.

regards
